assets/dev/js/frontend/handlers/column.js

```diff
@@ -1,5 +1,4 @@
 import BackgroundSlideshow from './background-slideshow';
 
-export default ( $scope ) => {
-	elementorFrontend.elementsHandler.addHandler( BackgroundSlideshow, { $element: $scope } );
-};
+export const ColumnHandlers = [	BackgroundSlideshow ];
```

Comment:
1. Since it's not a class, start it with a lower case letter (`columnHandlers`). Here and in the import.
2. Change the space at the beginning of the array.